browser/ui/views/profiles/brave_profile_menu_view.h

```diff
@@ -13,12 +13,12 @@ class BraveProfileMenuView : public ProfileMenuView {
   BraveProfileMenuView(const BraveProfileMenuView&) = delete;
   BraveProfileMenuView& operator=(const BraveProfileMenuView&) = delete;
 
- private:
   friend class ProfileMenuView;
 
   using ProfileMenuView::ProfileMenuView;
   ~BraveProfileMenuView() override = default;
 
+ private:
```

Comment:
?

Reply:
My intention was only to make the destructor public, otherwise the coordinator can't instantiate this class. Fixed in e34a7359213336bdd571c505c4b10c948ca67002.